node/src/reactor.rs

```diff
@@ -757,13 +757,15 @@ impl Runner<InitializerReactor> {
 
         info!("reactor main loop is ready");
 
+        let event_metrics_min_delay = Duration::from_secs(30);
+        let now = Instant::now();
         Ok(Runner {
             scheduler,
             reactor,
             event_count: 0,
             metrics: RunnerMetrics::new(&registry)?,
-            last_metrics: Instant::now(),
-            event_metrics_min_delay: Duration::from_secs(30),
+            last_metrics: now.checked_sub(event_metrics_min_delay).unwrap_or(now),
```

Comment:
A comment explaining why we're doing this subtraction would help.  Although this avoids the special case of `|| self.event_count == 0`, the special case at least made it completely obvious what was happening :)

Reply:
:+1: I'll add it!